src/components/routes/homepage/MediaCard.js

```diff
@@ -51,6 +57,8 @@ const useStyles = makeStyles(theme => ({
 
 export const MediaCard = ({ event }) => {
   const [elevation, setElevation] = useState(1);
+  const [isLiked, setLiked] = useState(false);
```

Comment:
I would recommend doing `[isLiked, setIsLiked]` for better clarity when used, despite how weird it might sound in english

Reply:
notedd